release_scripts/broad_rgp_release.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+gcloud storage --project=broad-rgp cp --recursive \
```

Comment:
Did you mean to use `--billing-project` here?

Reply:
Yes! I had a patch branch ready to go